### PR TITLE
Specify `rel="noopener"` for external links.

### PR DESCRIPTION
**Pull request by XhmikosR**
See here for an explanation: https://web.dev/external-anchors-use-rel-noopener/

Also, move `rel` and `target` attributes at the end.

Refs #331 

EDIT: needs a close review in case I missed something, although I did use a couple of regex's.

### CODE REVIEW

File: src/views/components/collection-links.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 	{{#each links}}
 	<div class="collection-link{{#if ~/combined && !@index && links.length > 1}} combined{{/if}}">
-		<a target="_blank" href="{{text}}">{{html}}</a>
+		<a href="{{text}}" rel="noopener" target="_blank">{{html}}</a>
```
Comment (XhmikosR) [inline src/views/components/collection-links.html:6]: Does this point to an external URL? If not we can drop the noopener addition

Comment (MartinKolarik) [reply src/views/components/collection-links.html:6]: No, that's our page.

**Comment by MartinKolarik** [comment]
> Also, move rel and target attributes at the end.

Please revert this and always put it first.

**Comment by XhmikosR** [comment]
Sorry, not going to revert it. It's illogical to me.

**Pull request closed without merging**